File: PooCC01CNetz/Semana03/Python/Ejercicios2/MateDos.py
from OperacionesVectores3D import *
from Vector3D import *
class MateDos:
    def main():
        ov = OperacionesVectores3D()
        a = Vector3D(2, 4, 5)
        b = Vector3D(-1, 1, 2)
        # Calcula el producto punto de dos vectores
        res = ov.calculaProductoPunto(a, b)
        print("Resultado producto punto de a y b: " + str(res))
        # Calcula el producto cruz de dos vectores
        c = ov.calculaProductoCruz(a, b)
        print("Componente X: " + str(c.getRefX()))
        print("Componente Y: " + str(c.getRefY()))
        print("Componente Z: " + str(c.getRefZ()))
    
    main()

File: PooCC01CNetz/Semana03/Java/Ejercicios2/MateDos.java
public class MateDos {
	public static void main(String []args) {
		OperacionesVectores3D ov = new OperacionesVectores3D();
		Vector3D a = new Vector3D(2, 4, 5);
		Vector3D b = new Vector3D(-1, 1, 2);
		//Calcula el producto punto de dos vectores
		double res = ov.calculaProductoPunto(a, b);
		System.out.println("Resultado producto punto de a y b: " + res);
		//Calcula el producto cruz de dos vectores
		Vector3D c = ov.calculaProductoCruz(a, b);
		System.out.println("Componente X: " + c.getRefX());
		System.out.println("Componente Y: " + c.getRefY());
		System.out.println("Componente Z: " + c.getRefZ());	
	}
}